sys/linux/test/landlock_ptrace

```diff
@@ -0,0 +1,26 @@
+# Creates independent Landlock hierarchies.
+
+r0 = fork()
```

Comment:
I guess I need to add `# requires: -arch=arm64 -arch=riscv64` because `fork()` doesn't exist for these architectures right?
Maybe something like `threaded=0` too?

Reply:
For `threaded=0` it's easier to answer.
Run:
```
make runtest
bin/syz-runtest -config manager.cfg -tests landlock_ptrace
```
And see what combinations pass/fail, and based on that you disabled what's failing (if it's expected).

Reply:
For other arches it's harder because we don't run these tests continuously on all arches. So if you add the following, I would say it's fine for now.
```
# fork is not present on these arches.
requires: -arch=arm64 -arch=riscv64
```

Reply:
syz-runtest pass for all tests now!